src/js/components/Peak/PeakStatistics.tsx

```diff
@@ -10,22 +12,30 @@ const PeakStatistics = ({ peakId }: { peakId: string }) => {
   const [firstConqueror, setFirstConqueror] = useState<User | undefined>(
     undefined
   );
+  const [lastConqueror, setLastConqueror] = useState<User | undefined>(
```

Comment:
Musimy to podmienić na jeden obiekt statystyk, ponieważ wygląda to źle

Reply:
Nie planowałam tak tego zostawić, ale czekam aż na backendzie już będzie to poprawione na jeden endpoint i wtedy to zmienię.

Reply:
Jasne, jasne zostawmy TODO i na teraz to jest ok